src/Glue.elm

```diff
@@ -130,29 +130,43 @@ poly :
     , set : subModel -> model -> model
     , init : ( subModel, Cmd msg )
     , update : subMsg -> subModel -> ( subModel, Cmd msg )
-    , view : subModel -> Html msg
     , subscriptions : subModel -> Sub msg
     }
     -> Glue model subModel msg subMsg
-poly { get, set, init, update, view, subscriptions } =
+poly { get, set, init, update, subscriptions } =
     Glue
         { model = set
         , init = init
         , update =
             \subMsg model ->
                 get model
                     |> update subMsg
-        , view =
-            \model ->
-                get model
-                    |> view
+        , view = \model -> text ""
         , subscriptions =
             \model ->
                 get model
                     |> subscriptions
         }
 
 
+{-| Helper for add custom view when using [`poly`](#poly) or [`simple`](#simple) constructor.
+
+
+
+**Interface:**
+
+```
+withView :
+    (model -> Html msg)
+    -> Glue model subModel msg subMsg
+    -> Glue model subModel msg subMsg
+```
+-}
+withView : (model -> Html msg) -> Glue model subModel msg subMsg -> Glue model subModel msg subMsg
+withView view (Glue data) =
+    Glue { data | view = view }
```

Comment:
This implementation won't be possible. There won't be `view` in `Glue`'s type record anymore - you won't be able to store this function anywhere.